doc/mediasdk-man.md

```diff
@@ -9424,6 +9424,8 @@ The `PlatformCodeName` enumerator itemizes Intel® microarchitecture code names.
 `MFX_PLATFORM_JASPERLAKE`   | Jasper Lake
 `MFX_PLATFORM_ELKHARTLAKE`  | Elkhart Lake
 `MFX_PLATFORM_TIGERLAKE`    | Tiger Lake
+`MFX_PLATFORM_ROCKETLAKE`   | Rocket Lake
+`MFX_PLATFORM_ALDERLAKE_S`  | Alder Lake S
```

Comment:
Adjust line 9444 below, please:
```
SDK API **TBD** adds `MFX_PLATFORM_ROCKETLAKE`, `MFX_PLATFORM_ALDERLAKE_S`, `MFX_PLATFORM_LAKEFIELD`.
```

Reply:
This is not done: update change history at line 9444.

Reply:
sorry, some error occur in last push and i missed the info. now it should be done as you request. thanks a lot.